cin_validator/rules/cin2022_23/rule_8875.py

```diff
@@ -0,0 +1,81 @@
+"""
+Rule number: 8875
+Module: Section 47
+Rule details: Where present, the <DateOfInitialCPC> (N00110) must not be a Saturday, Sunday
+Rule message: The Date of Initial Child Protection Conference cannot be a weekend
+
+"""
+from typing import Mapping
+
+import pandas as pd
+
+from cin_validator.rule_engine import (
+    CINTable,
+    IssueLocator,
+    RuleContext,
+    rule_definition,
+)
+from cin_validator.test_engine import run_rule
+
+# Get tables and columns of interest from the CINTable object defined in rule_engine/__api.py
+
+Section47 = CINTable.Section47
+DateOfInitialCPC = Section47.DateOfInitialCPC
+
+# define characteristics of rule
+@rule_definition(
+    code=8875,
+    module=CINTable.Section47,
+    message="The Date of Initial Child Protection Conference cannot be a weekend",
+    affected_fields=[DateOfInitialCPC],
+)
+def validate(
+    data_container: Mapping[CINTable, pd.DataFrame], rule_context: RuleContext
+):
+    df = data_container[Section47]
+
+    # <DateOfInitialCPC> should not be a Saturday, Sunday
+    # .weekday() returns the integer value for each day (0-6) with weekends being 5 and 6
+    failing_indices = df[df[DateOfInitialCPC].dt.weekday >= 5].index
```

Comment:
of course datetime has a nice fix for this, without thinking I assumed finding weekdays would be miserable